## 82% of end-end tests have passed
```
Total executed tests: 51
Failed tests: 7
Expected to fail tests: 2
Passed tests: 42
```
```
IDs of failed tests: 727230,702809,702733,704613,702957,727229,702958 
```
```
IDs of expected to fail tests: 703495,727232 
```
<h3>Failed tests (7)</h3><details><summary>Click to expand</summary><br/><li><a href="https://ci.status.im/job/status-mobile/job/e2e/job/status-app-prs-rerun/parambuild/?BRANCH_NAME=chore/remove-hot-reload-for-theme-change&PR_ID=20663&APK_NAME=20663.apk&TR_CASE_IDS=727230,702809,702733,704613,702957,727229,702958">Rerun failed tests</a></li><br/><h4>Class TestDeepLinksOneDevice:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3599539">test_links_open_universal_links_from_chat</a>, id: 704613 </b></td></tr><tr><td><p><blockquote><div>Device 1: Looking for chat: 'general' </div><div>Device 1: Click until `ChatMessageInput` by `accessibility id`: `chat-message-input` will be presented </div></blockquote></p>

```
critical/test_deep_and_universal_links.py:71: in test_links_open_universal_links_from_chat
    self.errors.verify_no_errors()
base_test_case.py:190: in verify_no_errors
    pytest.fail('\n '.join([self.errors.pop(0) for _ in range(len(self.errors))]))
 Community 'token-gated community e2e' was not requested to join by the url https://status.app/c/G0IAAGS9TbI9SrSPbDPplfBUeBBOIWhFRhGIKVlxntAg5CSvsJw4jurb636UKo7Lm2gTUS_M#zQ3shm7cS2MX6K1iinjNhiw88g1WH6fbE8rvnMHBq4RhznkWc
E    Community 'open community' was not requested to join by the url https://status.app/c/G0UAAMTyNsn2QZDEG0EXftOl8pOEfwEBOOSA_YTfIk85xmADDgINGmxpUHAXzK36bN0fK42Xf4YD2yjPk1z2pbFwFw==#zQ3shZU9PNP6QHwEmqoh5mQqjCMDsfZYnpY4BEC8hom7KoNEz
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/4e60192b4fca4fe0b0a09d8ead96df7c?auth=c06867026ace8e8ee7a0fec5204c76e4#52">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestWalletMultipleDevice:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3599542">test_wallet_send_asset_from_drawer</a>, id: 727230 </b></td></tr><tr><td><p><blockquote><div>Device 2: Tap on found: Button </div><div>Device 2: Find `ChatsTab` by `accessibility id`: `chats-stack-tab` </div></blockquote></p>

```
critical/test_wallet.py:154: in test_wallet_send_asset_from_drawer
    self.wallet_2.chats_tab.click()
../views/base_element.py:90: in click
    element = self.find_element()
../views/base_element.py:79: in find_element
    raise NoSuchElementException(
 Device 2: ChatsTab by accessibility id: `chats-stack-tab` is not found on the screen; For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
```

<br/><br/></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3599541">test_wallet_send_eth</a>, id: 727229 </b></td></tr><tr><td><p><blockquote><div>Device 1: Find `Button` by `accessibility id`: `top-bar` </div><div>Device 2: Find `Button` by `accessibility id`: `top-bar` </div></blockquote></p>

```
critical/test_wallet.py:147: in test_wallet_send_eth
    self.errors.verify_no_errors()
base_test_case.py:190: in verify_no_errors
    pytest.fail('\n '.join([self.errors.pop(0) for _ in range(len(self.errors))]))
 Sender balance is not updated on Etherscan, it is 0.0 but expected to be -0.0001
E    Receiver balance is not updated on Etherscan, it is 0.0026 but expected to be 0.0027
E    Eth amount in the senders wallet is 0.2001 but should be 0.2
E    Eth amount in the receivers wallet is 0.0026 but should be 0.0027
E    The last transaction is not listed in activity for the sender, expected timestamp is Today 11:59 AM
E    The last transaction is not listed in activity for the receiver, expected timestamp is Today 11:59 AM
```

<br/><br/></td></tr></tbody></table><h4>Class TestActivityMultipleDevicePRTwo:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3599526">test_activity_center_mentions</a>, id: 702957 </b></td></tr><tr><td><p><blockquote><div>Device 2: Find `Text` by `xpath`: `//android.view.ViewGroup[@content-desc='chat-item']//android.widget.TextView[contains(@text,'https://status.app/c/')]` </div><div>Device 2: Wait for element `Button` for max 120s and click when it is available </div></blockquote></p>

```
Test setup failed: activity_center/test_activity_center.py:414: in prepare_devices
    self.community_2.join_community()
../views/chat_view.py:420: in join_community
    self.join_button.wait_and_click(120)
../views/base_element.py:100: in wait_and_click
    self.wait_for_visibility_of_element(sec)
../views/base_element.py:147: in wait_for_visibility_of_element
    raise TimeoutException(
 Device 2: Button by accessibility id:`show-request-to-join-screen-button` is not found on the screen after wait_for_visibility_of_element
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/a6d2145dade34d21b2c2f7225adec89d?auth=a4d8fe723c87e59773a12df1ae95dd68">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/02c2ffe5d1524993bde0b6ab52473b79?auth=7ac811b162e12fa9b10812f8a83e2f71">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3599527">test_activity_center_admin_notification_accept_swipe</a>, id: 702958 </b></td></tr><tr><td>

```
Test setup failed: activity_center/test_activity_center.py:414: in prepare_devices
    self.community_2.join_community()
../views/chat_view.py:420: in join_community
    self.join_button.wait_and_click(120)
../views/base_element.py:100: in wait_and_click
    self.wait_for_visibility_of_element(sec)
../views/base_element.py:147: in wait_for_visibility_of_element
    raise TimeoutException(
 Device 2: Button by accessibility id:`show-request-to-join-screen-button` is not found on the screen after wait_for_visibility_of_element
```

<br/><br/></td></tr></tbody></table><h4>Class TestOneToOneChatMultipleSharedDevicesNewUi:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3599497">test_1_1_chat_text_message_delete_push_disappear</a>, id: 702733 </b></td></tr><tr><td>

```
critical/chats/test_1_1_public_chats.py:469: in test_1_1_chat_text_message_delete_push_disappear
    if not self.chat_2.chat_message_input.is_element_displayed():
../views/base_element.py:221: in is_element_displayed
    return self.wait_for_visibility_of_element(sec, ignored_exceptions=ignored_exceptions)
../views/base_element.py:145: in wait_for_visibility_of_element
    .until(expected_conditions.visibility_of_element_located((self.by, self.locator)))
../../../../status-app-prs@tmp/venv/lib/python3.10/site-packages/selenium/webdriver/support/wait.py:86: in until
    value = method(self._driver)
../../../../status-app-prs@tmp/venv/lib/python3.10/site-packages/selenium/webdriver/support/expected_conditions.py:152: in _predicate
    return _element_if_visible(driver.find_element(*locator))
../../../../status-app-prs@tmp/venv/lib/python3.10/site-packages/appium/webdriver/webdriver.py:409: in find_element
    return self.execute(RemoteCommand.FIND_ELEMENT, {'using': by, 'value': value})['value']
../../../../status-app-prs@tmp/venv/lib/python3.10/site-packages/selenium/webdriver/remote/webdriver.py:345: in execute
    self.error_handler.check_response(response)
../../../../status-app-prs@tmp/venv/lib/python3.10/site-packages/appium/webdriver/errorhandler.py:122: in check_response
    raise exception_class(msg=message, stacktrace=format_stacktrace(stacktrace))
 An unknown server-side error occurred while processing the command. Original error: 'POST /element' cannot be proxied to UiAutomator2 server because the instrumentation process is not running (probably crashed). Check the server log and/or the logcat output for more details
E   Stacktrace:
E   UnknownError: An unknown server-side error occurred while processing the command. Original error: 'POST /element' cannot be proxied to UiAutomator2 server because the instrumentation process is not running (probably crashed). Check the server log and/or the logcat output for more details
E       at UIA2Proxy.command (/mnt/sauce/appium/appium-v2.0.0/packages/base-driver/lib/jsonwp-proxy/proxy.js:355:13)
E       at processTicksAndRejections (node:internal/process/task_queues:95:5)
E       at AndroidUiautomator2Driver.helpers.doFindElementOrEls (/mnt/sauce/appium/appium-v2.0.0/appium_home/linux/node_modules/appium-uiautomator2-driver/lib/commands/find.js:30:10)
E       at doFind (/mnt/sauce/appium/appium-v2.0.0/appium_home/linux/node_modules/appium-uiautomator2-driver/node_modules/appium-android-driver/lib/commands/find.js:36:17)
E       at wrappedCondFn (/mnt/sauce/appium/appium-v2.0.0/packages/base-driver/lib/basedriver/commands/timeout.ts:137:14)
E       at spin (/mnt/sauce/appium/appium-v2.0.0/node_modules/asyncbox/lib/asyncbox.js:221:20)
E       at waitForCondition (/mnt/sauce/appium/appium-v2.0.0/node_modules/asyncbox/lib/asyncbox.js:238:10)
E       at AndroidUiautomator2Driver.implicitWaitForCondition (/mnt/sauce/appium/appium-v2.0.0/packages/base-driver/lib/basedriver/commands/timeout.ts:139:12)
E       at AndroidUiautomator2Driver.findElOrEls (/mnt/sauce/appium/appium-v2.0.0/appium_home/linux/node_modules/appium-uiautomator2-driver/node_modules/appium-android-driver/lib/commands/find.js:60:5)
E       at AndroidUiautomator2Driver.findElOrElsWithProcessing (/mnt/sauce/appium/appium-v2.0.0/packages/base-driver/lib/basedriver/commands/find.ts:60:12)
E       at AndroidUiautomator2Driver.findElement (/mnt/sauce/appium/appium-v2.0.0/packages/base-driver/lib/basedriver/commands/find.ts:75:12)
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/e12ceabb25a4434db9307e21b7624593?auth=1579aa8c55b4e7ca4c37e3f001496107#554">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/e46f758e621e420d9fd80018274cf240?auth=1852f80071c35c64897b3580f69fb3c5#430">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestCommunityMultipleDeviceMergedTwo:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3599508">test_community_markdown_support</a>, id: 702809 </b></td></tr><tr><td><p><blockquote><div>Device 1: Looking for a message by text: quote reply (one row) </div><div>Device 2: Looking for a message by text: quote reply (one row) </div></blockquote></p>

```
critical/chats/test_public_chat_browsing.py:1009: in test_community_markdown_support
    self.errors.verify_no_errors()
base_test_case.py:190: in verify_no_errors
    pytest.fail('\n '.join([self.errors.pop(0) for _ in range(len(self.errors))]))
 bold text in asterics is not displayed with markdown in 1-1 chat for the recipient (device 2)
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/17c4c80e6d384f2aa2607ee71c63f72a?auth=d775469af69fcfdb4a86021e87b3a2fe#184">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/b89c1f30f18b480fadf7096e63e6cd6b?auth=e25192653481e4608827f8ff063d2eac#173">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table></details><h3>Expected to fail tests (2)</h3><details><summary>Click to expand</summary><br/><br/><h4>Class TestWalletOneDevice:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3599544">test_wallet_add_remove_watch_only_account</a>, id: 727232 </b></td></tr><tr><td><p><blockquote><div>Device 1: Find `EditBox` by `accessibility id`: `add-address-to-watch` </div><div>Device 1: Type `0x8d2413447ff297d30bdc475f6d5cb00254685aae` to `EditBox` </div></blockquote></p>

```
critical/test_wallet.py:237: in test_wallet_add_remove_watch_only_account
    self.wallet_view.add_watch_only_account(address=address_to_watch, account_name=new_account_name)
../views/wallet_view.py:150: in add_watch_only_account
    self.account_has_activity_label.wait_for_visibility_of_element()
../views/base_element.py:147: in wait_for_visibility_of_element
    raise TimeoutException(
 Device 1: Text by accessibility id:`account-has-activity` is not found on the screen after wait_for_visibility_of_element 
```

<b>[[Missing networks in account address, https://github.com/status-im/status-mobile/issues/20166]]</b><br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/8ee486ae6d404b73afadd06637f7cc67?auth=d987d64e6f03988af473e974e505f9f3#102">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestGroupChatMultipleDeviceMergedNewUI:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3599535">test_group_chat_mute_chat</a>, id: 703495 </b></td></tr><tr><td><p><blockquote><div># STEP: Change device time so chat will be unmuted by timer</div><div>Device 2: Long press on `ChatElement` </div></blockquote></p>

```
critical/chats/test_group_chat.py:495: in test_group_chat_mute_chat
    self.errors.verify_no_errors()
base_test_case.py:190: in verify_no_errors
    pytest.fail('\n '.join([self.errors.pop(0) for _ in range(len(self.errors))]))
 Chat is still muted after timeout 
```

<b>[[Chat is not unmuted after expected time: https://github.com/status-im/status-mobile/issues/19627]]</b><br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/ce4f00e9938e439eaafb2e1c2f7faf0a?auth=b2f6089acce4f700352013b95e6f9337#473">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/9a672cfb61cc4d23af21e78796a2fc8c?auth=b969e02ddabfabea60e2af1c40feef15#273">Steps, video, logs</a></li></ul></p><p>Device 3:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/3a9cf2aaa5c2446abe68c08c0599a840?auth=58d2f15953401b0f0e07cb5ebeab99b4#242">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table></details><h3>Passed tests (42)</h3><details><summary>Click to expand</summary><br/><br/><h4>Class TestCommunityOneDeviceMerged:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3599529">test_restore_multiaccount_with_waku_backup_remove_switch</a>, id: 703133 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/c9f49371f5a24c288c1649eabfbd9131?auth=167e248dca3c0bb71c8d729498f74f05#229">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3599498">test_community_copy_and_paste_message_in_chat_input</a>, id: 702742 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/c9f49371f5a24c288c1649eabfbd9131?auth=167e248dca3c0bb71c8d729498f74f05#76">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3599537">test_community_discovery</a>, id: 703503 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/c9f49371f5a24c288c1649eabfbd9131?auth=167e248dca3c0bb71c8d729498f74f05#354">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>4. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3599522">test_community_undo_delete_message</a>, id: 702869 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/c9f49371f5a24c288c1649eabfbd9131?auth=167e248dca3c0bb71c8d729498f74f05#105">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>5. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3599517">test_community_navigate_to_channel_when_relaunch</a>, id: 702846 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/c9f49371f5a24c288c1649eabfbd9131?auth=167e248dca3c0bb71c8d729498f74f05#52">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>6. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3599533">test_community_mute_community_and_channel</a>, id: 703382 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/c9f49371f5a24c288c1649eabfbd9131?auth=167e248dca3c0bb71c8d729498f74f05#124">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestGroupChatMultipleDeviceMergedNewUI:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3599496">test_group_chat_pin_messages</a>, id: 702732 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/ce4f00e9938e439eaafb2e1c2f7faf0a?auth=b2f6089acce4f700352013b95e6f9337#353">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/9a672cfb61cc4d23af21e78796a2fc8c?auth=b969e02ddabfabea60e2af1c40feef15#230">Steps, video, logs</a></li></ul></p><p>Device 3:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/3a9cf2aaa5c2446abe68c08c0599a840?auth=58d2f15953401b0f0e07cb5ebeab99b4#228">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3599532">test_group_chat_send_image_save_and_share</a>, id: 703297 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/ce4f00e9938e439eaafb2e1c2f7faf0a?auth=b2f6089acce4f700352013b95e6f9337#280">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/9a672cfb61cc4d23af21e78796a2fc8c?auth=b969e02ddabfabea60e2af1c40feef15#168">Steps, video, logs</a></li></ul></p><p>Device 3:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/3a9cf2aaa5c2446abe68c08c0599a840?auth=58d2f15953401b0f0e07cb5ebeab99b4#141">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3599531">test_group_chat_reactions</a>, id: 703202 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/ce4f00e9938e439eaafb2e1c2f7faf0a?auth=b2f6089acce4f700352013b95e6f9337#173">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/9a672cfb61cc4d23af21e78796a2fc8c?auth=b969e02ddabfabea60e2af1c40feef15#106">Steps, video, logs</a></li></ul></p><p>Device 3:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/3a9cf2aaa5c2446abe68c08c0599a840?auth=58d2f15953401b0f0e07cb5ebeab99b4#86">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>4. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3599506">test_group_chat_join_send_text_messages_push</a>, id: 702807 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/ce4f00e9938e439eaafb2e1c2f7faf0a?auth=b2f6089acce4f700352013b95e6f9337#156">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/9a672cfb61cc4d23af21e78796a2fc8c?auth=b969e02ddabfabea60e2af1c40feef15#78">Steps, video, logs</a></li></ul></p><p>Device 3:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/3a9cf2aaa5c2446abe68c08c0599a840?auth=58d2f15953401b0f0e07cb5ebeab99b4#78">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>5. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3599507">test_group_chat_offline_pn</a>, id: 702808 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/ce4f00e9938e439eaafb2e1c2f7faf0a?auth=b2f6089acce4f700352013b95e6f9337#320">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/9a672cfb61cc4d23af21e78796a2fc8c?auth=b969e02ddabfabea60e2af1c40feef15#203">Steps, video, logs</a></li></ul></p><p>Device 3:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/3a9cf2aaa5c2446abe68c08c0599a840?auth=58d2f15953401b0f0e07cb5ebeab99b4#201">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestDeepLinksOneDevice:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3599500">test_links_deep_links</a>, id: 702775 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/4e60192b4fca4fe0b0a09d8ead96df7c?auth=c06867026ace8e8ee7a0fec5204c76e4#215">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestOneToOneChatMultipleSharedDevicesNewUiTwo:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3599504">test_1_1_chat_delete_via_long_press_relogin</a>, id: 702784 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/a06e5eca0dda422aa30d03fe250bc56f?auth=25307619560e9bcecef55c8ee6a80d73#171">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/def860f820134c658d70be25446308d9?auth=a4d9bf08b7f58b96b02ca86c16e64ea5#125">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3599503">test_1_1_chat_is_shown_message_sent_delivered_from_offline</a>, id: 702783 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/a06e5eca0dda422aa30d03fe250bc56f?auth=25307619560e9bcecef55c8ee6a80d73#95">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/def860f820134c658d70be25446308d9?auth=a4d9bf08b7f58b96b02ca86c16e64ea5#84">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3599536">test_1_1_chat_mute_chat</a>, id: 703496 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/a06e5eca0dda422aa30d03fe250bc56f?auth=25307619560e9bcecef55c8ee6a80d73#105">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/def860f820134c658d70be25446308d9?auth=a4d9bf08b7f58b96b02ca86c16e64ea5#112">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestActivityMultipleDevicePR:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3599524">test_activity_center_reply_read_unread_delete_filter_swipe</a>, id: 702947 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/ca57cc24b42d4d32acfd4215745a656f?auth=f15c899caaa7cd56daa0200c292e7579#128">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/af4cdf0cf134416b9021785a525bb921?auth=deba9a7c16051725fbe9c668bf6ca0f8#98">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestOneToOneChatMultipleSharedDevicesNewUi:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3599502">test_1_1_chat_emoji_send_reply_and_open_link</a>, id: 702782 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/e12ceabb25a4434db9307e21b7624593?auth=1579aa8c55b4e7ca4c37e3f001496107#147">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/e46f758e621e420d9fd80018274cf240?auth=1852f80071c35c64897b3580f69fb3c5#119">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3599509">test_1_1_chat_push_emoji</a>, id: 702813 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/e12ceabb25a4434db9307e21b7624593?auth=1579aa8c55b4e7ca4c37e3f001496107#413">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/e46f758e621e420d9fd80018274cf240?auth=1852f80071c35c64897b3580f69fb3c5#259">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3599499">test_1_1_chat_non_latin_messages_stack_update_profile_photo</a>, id: 702745 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/e12ceabb25a4434db9307e21b7624593?auth=1579aa8c55b4e7ca4c37e3f001496107#355">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/e46f758e621e420d9fd80018274cf240?auth=1852f80071c35c64897b3580f69fb3c5#209">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>4. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3599520">test_1_1_chat_edit_message</a>, id: 702855 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/e12ceabb25a4434db9307e21b7624593?auth=1579aa8c55b4e7ca4c37e3f001496107#462">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/e46f758e621e420d9fd80018274cf240?auth=1852f80071c35c64897b3580f69fb3c5#290">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>5. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3599534">test_1_1_chat_send_image_save_and_share</a>, id: 703391 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/e12ceabb25a4434db9307e21b7624593?auth=1579aa8c55b4e7ca4c37e3f001496107#493">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/e46f758e621e420d9fd80018274cf240?auth=1852f80071c35c64897b3580f69fb3c5#370">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>6. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3599495">test_1_1_chat_pin_messages</a>, id: 702731 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/e12ceabb25a4434db9307e21b7624593?auth=1579aa8c55b4e7ca4c37e3f001496107#209">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/e46f758e621e420d9fd80018274cf240?auth=1852f80071c35c64897b3580f69fb3c5#164">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>7. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3599494">test_1_1_chat_message_reaction</a>, id: 702730 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/e12ceabb25a4434db9307e21b7624593?auth=1579aa8c55b4e7ca4c37e3f001496107#95">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/e46f758e621e420d9fd80018274cf240?auth=1852f80071c35c64897b3580f69fb3c5#84">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestCommunityMultipleDeviceMerged:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3599530">test_community_several_images_send_reply</a>, id: 703194 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/9c1af6debbc04a9dbbc4201584a97459?auth=4e918e3f49245dccb69ff8f593d0ad37#233">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/6724228e86624ddf97e2a8add27ba961?auth=9dbd02fc09722717b81660c25866b945#167">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3599521">test_community_one_image_send_reply</a>, id: 702859 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/9c1af6debbc04a9dbbc4201584a97459?auth=4e918e3f49245dccb69ff8f593d0ad37#258">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/6724228e86624ddf97e2a8add27ba961?auth=9dbd02fc09722717b81660c25866b945#228">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3599512">test_community_emoji_send_copy_paste_reply</a>, id: 702840 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/9c1af6debbc04a9dbbc4201584a97459?auth=4e918e3f49245dccb69ff8f593d0ad37#306">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/6724228e86624ddf97e2a8add27ba961?auth=9dbd02fc09722717b81660c25866b945#288">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>4. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3599528">test_community_mark_all_messages_as_read</a>, id: 703086 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/9c1af6debbc04a9dbbc4201584a97459?auth=4e918e3f49245dccb69ff8f593d0ad37#552">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/6724228e86624ddf97e2a8add27ba961?auth=9dbd02fc09722717b81660c25866b945#470">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>5. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3599523">test_community_contact_block_unblock_offline</a>, id: 702894 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/9c1af6debbc04a9dbbc4201584a97459?auth=4e918e3f49245dccb69ff8f593d0ad37#425">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/6724228e86624ddf97e2a8add27ba961?auth=9dbd02fc09722717b81660c25866b945#396">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>6. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3599540">test_community_edit_delete_message_when_offline</a>, id: 704615 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/9c1af6debbc04a9dbbc4201584a97459?auth=4e918e3f49245dccb69ff8f593d0ad37#587">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/6724228e86624ddf97e2a8add27ba961?auth=9dbd02fc09722717b81660c25866b945#496">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>7. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3599511">test_community_message_delete</a>, id: 702839 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/9c1af6debbc04a9dbbc4201584a97459?auth=4e918e3f49245dccb69ff8f593d0ad37#200">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/6724228e86624ddf97e2a8add27ba961?auth=9dbd02fc09722717b81660c25866b945#138">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>8. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3599510">test_community_message_send_check_timestamps_sender_username</a>, id: 702838 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/9c1af6debbc04a9dbbc4201584a97459?auth=4e918e3f49245dccb69ff8f593d0ad37#158">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/6724228e86624ddf97e2a8add27ba961?auth=9dbd02fc09722717b81660c25866b945#109">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>9. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3599515">test_community_links_with_previews_github_youtube_twitter_gif_send_enable</a>, id: 702844 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/9c1af6debbc04a9dbbc4201584a97459?auth=4e918e3f49245dccb69ff8f593d0ad37#330">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/6724228e86624ddf97e2a8add27ba961?auth=9dbd02fc09722717b81660c25866b945#305">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>10. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3599514">test_community_message_edit</a>, id: 702843 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/9c1af6debbc04a9dbbc4201584a97459?auth=4e918e3f49245dccb69ff8f593d0ad37#176">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/6724228e86624ddf97e2a8add27ba961?auth=9dbd02fc09722717b81660c25866b945#126">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>11. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3599513">test_community_unread_messages_badge</a>, id: 702841 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/9c1af6debbc04a9dbbc4201584a97459?auth=4e918e3f49245dccb69ff8f593d0ad37#395">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/6724228e86624ddf97e2a8add27ba961?auth=9dbd02fc09722717b81660c25866b945#366">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestWalletOneDevice:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3599543">test_wallet_add_remove_regular_account</a>, id: 727231 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/8ee486ae6d404b73afadd06637f7cc67?auth=d987d64e6f03988af473e974e505f9f3#38">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestCommunityMultipleDeviceMergedTwo:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3599525">test_community_hashtag_links_to_community_channels</a>, id: 702948 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/17c4c80e6d384f2aa2607ee71c63f72a?auth=d775469af69fcfdb4a86021e87b3a2fe#303">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/b89c1f30f18b480fadf7096e63e6cd6b?auth=e25192653481e4608827f8ff063d2eac#301">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3599505">test_community_mentions_push_notification</a>, id: 702786 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/17c4c80e6d384f2aa2607ee71c63f72a?auth=d775469af69fcfdb4a86021e87b3a2fe#135">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/b89c1f30f18b480fadf7096e63e6cd6b?auth=e25192653481e4608827f8ff063d2eac#115">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3599516">test_community_leave</a>, id: 702845 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/17c4c80e6d384f2aa2607ee71c63f72a?auth=d775469af69fcfdb4a86021e87b3a2fe#300">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/b89c1f30f18b480fadf7096e63e6cd6b?auth=e25192653481e4608827f8ff063d2eac#279">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>4. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3599538">test_community_join_when_node_owner_offline</a>, id: 703629 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/17c4c80e6d384f2aa2607ee71c63f72a?auth=d775469af69fcfdb4a86021e87b3a2fe#502">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/b89c1f30f18b480fadf7096e63e6cd6b?auth=e25192653481e4608827f8ff063d2eac#487">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestActivityCenterContactRequestMultipleDevicePR:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3599501">test_add_contact_field_validation</a>, id: 702777 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/274a63e8816c44b3b629ebd746eba334?auth=ccd4f81162042dbdf4b3db9bba7b4527#134">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/422cb1882e10478e9074cd91f963c4e0?auth=3715b8ac17d75c8e7cc3740a7c1a0775#206">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3599519">test_activity_center_contact_request_accept_swipe_mark_all_as_read</a>, id: 702851 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/274a63e8816c44b3b629ebd746eba334?auth=ccd4f81162042dbdf4b3db9bba7b4527#99">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/422cb1882e10478e9074cd91f963c4e0?auth=3715b8ac17d75c8e7cc3740a7c1a0775#113">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3599518">test_activity_center_contact_request_decline</a>, id: 702850 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/274a63e8816c44b3b629ebd746eba334?auth=ccd4f81162042dbdf4b3db9bba7b4527#56">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/422cb1882e10478e9074cd91f963c4e0?auth=3715b8ac17d75c8e7cc3740a7c1a0775#55">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table></details>

--- a/src/legacy/status_im/events.cljs
+++ b/src/legacy/status_im/events.cljs
@@ -108,8 +108,7 @@
     (when (and (multiaccounts.model/logged-in? db)
                (= current-theme-type status-im.constants/theme-type-system))
       {:profile.settings/switch-theme-fx
-       [(get-in db [:profile/profile :appearance])
-        (:view-id db) true]})))
+       [(get-in db [:profile/profile :appearance]) (:view-id db)]})))
 
 (defn- on-biometric-auth-fail
   [{:keys [code]}]

--- a/src/status_im/contexts/profile/login/events.cljs
+++ b/src/status_im/contexts/profile/login/events.cljs
@@ -83,8 +83,7 @@
                   [[:profile.settings/switch-theme-fx
                     [(or (:appearance settings)
                          constants/theme-type-dark)
-                     :shell-stack
-                     false]]
+                     :shell-stack]]
                    [:dispatch [:init-root :shell-stack]]
                    [:dispatch [:profile/show-testnet-mode-banner-if-enabled]]]))})))
 

--- a/src/status_im/contexts/profile/settings/effects.cljs
+++ b/src/status_im/contexts/profile/settings/effects.cljs
@@ -4,7 +4,6 @@
             [react-native.platform :as platform]
             [status-im.common.theme.core :as theme]
             [status-im.constants :as constants]
-            [status-im.setup.hot-reload :as hot-reload]
             [utils.re-frame :as rf]))
 
 (re-frame/reg-fx
@@ -20,7 +19,7 @@
 
 (re-frame/reg-fx
  :profile.settings/switch-theme-fx
- (fn [[theme-type view-id reload-ui?]]
+ (fn [[theme-type view-id]]
    (let [theme (if (or (= theme-type constants/theme-type-dark)
                        (and (= theme-type constants/theme-type-system)
                             (theme/device-theme-dark?)))
@@ -28,8 +27,4 @@
                  :light)]
      (theme/set-legacy-theme theme)
      (rf/dispatch [:theme/switch theme])
-     (rf/dispatch [:reload-status-nav-color view-id])
-     (when reload-ui?
-       (re-frame/dispatch [:dismiss-all-overlays])
-       (when js/goog.DEBUG
-         (hot-reload/reload))))))
+     (rf/dispatch [:reload-status-nav-color view-id]))))

--- a/src/status_im/contexts/profile/settings/events.cljs
+++ b/src/status_im/contexts/profile/settings/events.cljs
@@ -95,14 +95,14 @@
 (rf/reg-event-fx :profile.settings/change-appearance
  (fn [_ [theme]]
    {:fx [[:dispatch [:profile.settings/profile-update :appearance theme]]
-         [:profile.settings/switch-theme-fx [theme :appearance true]]]}))
+         [:profile.settings/switch-theme-fx [theme :appearance]]]}))
 
 (rf/reg-event-fx :profile.settings/switch-theme
  (fn [{:keys [db]} [theme view-id]]
    (let [theme (or theme
                    (get-in db [:profile/profile :appearance])
                    constants/theme-type-dark)]
-     {:fx [[:profile.settings/switch-theme-fx [theme view-id false]]]})))
+     {:fx [[:profile.settings/switch-theme-fx [theme view-id]]]})))
 
 (rf/reg-fx :profile.settings/get-profile-picture
  (fn [key-uid]

--- a/src/status_im/navigation/events.cljs
+++ b/src/status_im/navigation/events.cljs
@@ -123,16 +123,6 @@
                     (hide-bottom-sheet new-cofx)
                     {:show-bottom-sheet {:theme theme}}))))))
 
-(rf/defn dismiss-all-overlays
-  {:events [:dismiss-all-overlays]}
-  [_]
-  {:dispatch-n [[:hide-popover]
-                [:hide-visibility-status-popover]
-                [:hide-bottom-sheet]
-                [:bottom-sheet-hidden]
-                [:bottom-sheet/hide-old-navigation-overlay]
-                [:toasts/close-all-toasts]]})
-
 (rf/defn set-view-id
   {:events [:set-view-id]}
   [{:keys [db]} view-id]

